Skysim production

evevkovacs opened the pull request:
New pull request with the .git directories removed

aphearin commented:
Quite a large amount of code changes in this PR @evevkovacs - could you point me to the pieces you are most concerned with being reviewed?

evevkovacs commented:
This is the commit for all of Dan's code being used for the skysim5000 production pipeline. I don't think it needs a detailed review. I reworked Dan's scripts to handle a more automated production pipeline. This code is in lightcone_resample/params_lc_resamp/skysim5000/
make_param.sh.  I also cleaned up the old code and removed old directories of figures and plots. Mostly I think you should just take a quick look to get an idea of what is there and check if anything jumps out at you. The main code that gets run to do the matchup is lightcone_resample/lc_reasmple.py. I made some changes to this to make sure that the plots got named for their contents (instead of figxx.png) and got written to  a subdirectory od the output directory rather than a directory in lightcone_resample. This has all been tested and is working.
